index.html

```diff
@@ -687,31 +877,26 @@ <h2 class="subscribe-block__title title">
             <div class="search-container subscribe-wrapper">
               <!-- Search form -->
 
-              <form
-                class="search-container__search-form mail-form"
-                method="get"
-              >
+              <form class="search-container__search-form mail-form">
                 <input
                   class="mail-input"
                   type="text"
                   placeholder="infomail856@gmail.com"
                   name="text"
                   value=""
                 />
-              </form>
 
-              <!-- Subscribe button -->
-              <div class="subscribe-block__button">
-                <a class="buttons subscribe-button" href="#">
+                <!-- Subscribe button -->
+                <button type="submit" class="button subscribe-button">
                   Subscribe
-                  <img src="./src/icons/telegram.svg" alt="telegram icon" />
-                </a>
-              </div>
+                  <img class="telegram" src="./src/icons/telegram.svg" alt="" />
```

Comment:
```suggestion
                  <img class="send-icon" src="./src/icons/telegram.svg" alt="" />
```
This is not the Telegram logo!